app/components/forms/session-speaker-form.js

```diff
@@ -371,7 +371,7 @@ export default Component.extend(FormMixin, {
           rules      : [
             {
               type   : 'regExp',
-              value  : compulsoryProtocolValidUrlPattern,
+              value  : protocolLessValidUrlPattern,
```

Comment:
if you had to change the regex for website, won't you have to do it for twitter and facebook as well?